realwork/realwork/Dashboard/HomeBoard/MusicBarViewController.swift

```diff
@@ -45,5 +72,28 @@ class MusicBarViewController: UIViewController {
             }
             playPauseButton.setBackgroundImage(img, for: .normal)
         }
+        sendToWatch()
+    }
+}
+
+extension MusicBarViewController: WCSessionDelegate {
+
+  func sessionDidBecomeInactive(_ session: WCSession) {
+  }
+
+  func sessionDidDeactivate(_ session: WCSession) {
+  }
+
+  func session(_ session: WCSession, activationDidCompleteWith activationState: WCSessionActivationState, error: Error?) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 122 characters](https://app.codacy.com/manual/NathanDVT/Musico/pullRequest?prid=5444460)